collectors/service.go

```diff
@@ -42,6 +42,12 @@ var (
 		descServiceLabelsHelp,
 		descServiceLabelsDefaultLabels, nil,
 	)
+
+	descServiceSpecServiceType = prometheus.NewDesc(
+		"kube_service_spec_servicetype",
```

Comment:
@zouyee How about we change metric name to `kube_service_spec_type`. `servicetype` seems redundant.